gettsim/data/soz_vers_beitr.yaml

```diff
@@ -78,12 +78,15 @@ gkvbs_ag:
     2011-01-01:
       value: 0.073
       note:
+    2019-01-01:
+      value: 0.0785
+      note: Wieder-Einführung der Parität. Art. 2 G. v. 11.12.2018, BGBl. I S. 2387
```

Comment:
Why do we not update now the parameters for 2020? Wasn't the average Zusatzbeitrag 0.9 2019 and therefore 7.3 + 0.45 = 7.75 for 2019 and 7.85 for 2020?